src/utils/revurdering/revurderingUtils.ts

```diff
@@ -128,3 +137,20 @@ export const finnNesteRevurderingsteg = (
 
     return førsteIkkeVurderteSteg ?? RevurderingSteg.Oppsummering;
 };
+
+export const getAvsluttedeOgIkkeAvsluttedeRevurderinger = (
+    revurderinger: AbstraktRevurdering[]
+): { avsluttedeRevurderinger: AbstraktRevurdering[]; åpneRevurderinger: AbstraktRevurdering[] } => {
+    const avsluttedeRevurderinger: AbstraktRevurdering[] = [];
+    const åpneRevurderinger: AbstraktRevurdering[] = [];
+
+    revurderinger.forEach((r) => {
+        if (erAbstraktRevurderingAvsluttet(r)) {
+            avsluttedeRevurderinger.push(r);
+        } else {
+            åpneRevurderinger.push(r);
+        }
+    });
+
+    return { avsluttedeRevurderinger, åpneRevurderinger };
+};
```

Comment:
```suggestion
export const splittAvsluttedeOgÅpneRevurderinger = (
  revurderinger: AbstraktRevurdering[]
): { avsluttedeRevurderinger: AbstraktRevurdering[]; åpneRevurderinger: AbstraktRevurdering[] } =>
    pipe(
        revurderinger,
        A.partition(erAbstraktRevurderingAvsluttet),
        ({ left, right }) => ({
            avsluttedeRevurderinger: right,
            åpneRevurderinger: left
        })
    )
```
der `A` er `fp-ts/Array`.